packages/axes/src/inputType/WheelInput.ts

```diff
@@ -191,7 +203,9 @@ export class WheelInput implements InputType {
       throw new Error("Element to connect input does not exist.");
     }
     this._observer = observer;
-    element.addEventListener("wheel", this._onWheel);
+    element.addEventListener("wheel", this._onWheel, {
```

Comment:
Do I need to add passive?

Need to add passive in axes?

There are cases where users do preventDefault while using the wheel event.

There are cases where users do preventDefault while using the wheel event.

Reply:
I was concerned about warnings about non-passive event listeners, but considering `preventDefault` it seems right to keep passive as false.